Translated/Dialog/Somua/ssn_014020.csv

```diff
@@ -0,0 +1,10 @@
+name_7201#0,"""Somua"""
+ssn_014020_0001#0,"""Central City is at the forefront of our war with the DOLLS."""
+ssn_014020_0002#0,"""This is a place where many ARKS gather, so don't be afraid to ask someone for help."""
+ssn_014020_0101#0,"""Do you know of the \""Gigantix\""? The fearsome beasts that are said to appear during intense storms?"""
```

Comment:
I tweaked the punctuation on the end of this line, but looking at the next I think it needs a little more of a rewrite to get it to flow properly. The second sentence should probably be a more definitive statement of the rumours rather than a continuation of the question.

Reply:
oh i see what you mean. gave her some dank flow